src/main/java/org/example/Calculator.java

```diff
@@ -0,0 +1,59 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter operand1: ");
+        double operand1 = scanner.nextDouble();
+
+        System.out.print("Enter operator (+, -, x, /): ");
+        String operator = scanner.next();
+
+        System.out.print("Enter operand2: ");
+        double operand2 = scanner.nextDouble();
+        if (operand1 == 0 && operand2 == 0
+                && ("/".equals(operator) || "x".equals(operator)
+                || ":".equals(operator) || "*".equals(operator))){
```

Comment:
why don't you allow multiplication by zero?  from a mathematical point of view, this is a valid operation

Reply:
Checks for : and * are redundant as they are nit allowed by swich below